packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts

```diff
@@ -84,7 +81,7 @@ export class BasicTracerProvider implements TracerProvider {
     name: string,
     version?: string,
     options?: { schemaUrl?: string }
-  ): Tracer {
+  ): ApiTracer {
```

Comment:
Note: if there's time, I'll also follow-up with renaming the now internal tracer to `SdkTracer` to avoid having to do this and to have the code be more readable.

I've opted to not do this in this PR to keep the diff somewhat contained.